addons/vagrant/inventory/hosts

```diff
@@ -1671,3 +1671,329 @@ all:
             netmask: '255.255.255.0'
             ip_reg: '172.18.156.34'
             ip_iso: '172.18.157.34'
+      _abprasa:
```

Comment:
Must be to follow naming convention:
```suggestion
      _aprasad:
```

Reply:
and also in next lines.

Reply:
Like discussed it is more the akamai convention way.
Lets keep it